.github/CONTRIBUTING.md

```diff
@@ -216,9 +216,21 @@ Some notes:
 
 TBD - need to hash out a bit more of this process.
 
-## Tag A Release
+## Release Workflow
 
-TBD - need to hash out a bit more of this process. The most important bit here is probably that all tests must pass, and tags must use [semver](https://semver.org).
+With GitHub actions, our workflow for publishing new version to NPM are automatic! Here are the steps you need to take to ensure a release occurs:
+
+1. In a commit, use the [NPM `version` command](https://docs.npmjs.com/cli/version) to update the package to the next version:
```

Comment:
the `npm version` command creates a commit for you, no?

Reply:
Oh whoops you're right! Let me update that